src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -364,6 +351,19 @@ public void Initialize(InitializeOptions options = InitializeOptions.None, DateT
                         }
                     }
                 }, DatabaseShutdown);
+
+                try
+                {
+                    _notifyFeaturesChangedTask.Wait();
```

Comment:
We weren't waiting before, because it might take a lot of time and stall the loading of other databases

Reply:
This results in missing indexes after the database started.

Reply:
what do u mean by missing indexes? on the database record?

Reply:
The database was restored (smuggler). We start the database and the indexes aren't started (yet).
You are right, we cannot wait here. I'll fix the test.